site/content/en/topics/loadbalancer.md

```diff
@@ -27,7 +27,11 @@ Below is a list of annotations supported for Kubernetes services with type `Load
 | `service.beta.kubernetes.io/azure-allowed-service-tags`                         | List of allowed service tags                                                                                                           | Specify a list of allowed [service tags](https://docs.microsoft.com/en-us/azure/virtual-network/security-overview#service-tags) separated by comma.                                                                                                                                                                                                                                                                                                                         | v1.11.0 and later                                 |
 | `service.beta.kubernetes.io/azure-load-balancer-tcp-idle-timeout`               | TCP idle timeouts in minutes                                                                                                           | Specify the time, in minutes, for TCP connection idle timeouts to occur on the load balancer. Default and minimum value is 4. Maximum value is 30. Must be an integer.                                                                                                                                                                                                                                                                                                      | v1.11.4, v1.12.0 and later                        |
 | `service.beta.kubernetes.io/azure-pip-name`                                     | Name of PIP                                                                                                                            | Specify the PIP that will be applied to load balancer.                                                                                                                                                                                                                                                                                                                                                                                                                      | v1.16 and later                                   |
+| `service.beta.kubernetes.io/azure-pip-name-ipv4` | Name of IPv4 PIP | Specify the IPv4 PIP that will be applied to load balancer. | v1.27 and later |
```

Comment:
Should we deprecate `service.beta.kubernetes.io/azure-pip-name` or should we merge the usage between them?

Reply:
Good point! First, I don't think we can deprecate the old annotation because of compatibility according to past experience with LB IP.
Current code logic is "merge", check new annotation first then old one.

Reply:
If we can use the old one for ipv4 usage, we can get rid of `service.beta.kubernetes.io/azure-pip-name-ipv4` right?

Reply:
The old one is not for IPv4 only. If it is a single stack IPv6 cluster, the old annotation also can be used.

Reply:
Correct, but we can make it for ipv4 only when enabling dual stack.

Reply:
Understand your point. But this will break current behavior for IPv6 single stack cluster which I don't think we should. @feiskyer

Reply:
Is it possible to keep the logic for single stack as is and only change its usage for dual stack?

Reply:
would the new -ipv4 and -ipv6 annotations overwrite existing pip-name annotation? If so, it should be ok.

Reply:
Discussed offline, will follow @nilo19 's suggstion and use old pip name annotation for dualstack cluster's IPv4 pip. It relies on users correctly understanding the usage of two annotations.